packages/contracts-ops/contracts/Config.sol

```diff
@@ -122,6 +128,24 @@ abstract contract Config {
             );
     }
 
+    function replicaOfUpdgrade(string memory local, string memory replica)
```

Comment:
typo in function name

Reply:
wdyt rename `replica` to `remote` to follow convention with sdk more closely?

Reply:
meh